.github/workflows/build_image.yml

```diff
@@ -0,0 +1,52 @@
+name: Build image
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*"
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+env:
+  DOCKER_BUILDKIT: 1
```

Comment:
What is the purpose of defining `DOCKER_BUILDKIT`?
If your goal is to enable caching, define the option in `docker/build-push-action@v4`.

> Buildx will always build using the BuildKit engine and does not require DOCKER_BUILDKIT=1 environment variable for starting builds.

Ref: [docker/buildx: Docker CLI plugin for extended build capabilities with BuildKit](https://github.com/docker/buildx#building-with-buildx)
Ref: [Cache management with GitHub Actions | Docker Documentation](https://docs.docker.com/build/ci/github-actions/cache/)

Reply:
My goal is to use buildkit, but it not to enable caching.

Recent Docker Desktop uses buildkit engine and I want to use the same engine on CI.
I think caching is unnecessary for the repository yet.

---

> Buildx will always build using the BuildKit engine and does not require DOCKER_BUILDKIT=1 environment variable for starting builds.

Thank you. I'll remove the redundunt `DOCKER_BUILDKIT: 1` :-)